example-feature/example-feature-api-android/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+android {
+    namespace "com.duckduckgo.examplefeature.api"
+}
+
+dependencies {
+    implementation project(':navigation-api')
```

Comment:
Is this essential for all Android modules?

Reply:
Not strictly _essential_ for all api modules, but it's used in a lot (most?) of the existing ones already. Debatable. Will leave it for now, preferring to merge and iterate later.